cqproto/internal/plugin.proto

```diff
@@ -134,20 +133,15 @@ message GetProviderSchema {
   }
 }
 
-enum ConfigFormat {
-  HCL = 0;
-  YAML = 1;
-}
-
 message GetProviderConfig {
   message Request {
-    ConfigFormat format = 1;
+    // next available index: 2
   }
   message Response {
     string name = 1;
     string version = 2;
     bytes config = 3;
-    ConfigFormat format = 4;
```

Comment:
maybe keep this but mark it as deprecated

Reply:
In the response only?